packages/scene-composer/stories/SceneComposer.stories.tsx

```diff
@@ -144,13 +144,13 @@ const knobsConfigurationDecorator = [
     args.config = {
       featureConfig: {
         [COMPOSER_FEATURES.SceneHierarchyRedesign]: true, // New Scene Hierarchy Panel
-        [COMPOSER_FEATURES.SceneHierarchySearch]: false, // Entity Search
-        [COMPOSER_FEATURES.SceneHierarchyMultiSelect]: false, // MultiSelect disabled, not sure if we will support this.
-        [COMPOSER_FEATURES.SceneHierarchyReorder]: false, // Drag/Drop Reordering
-        [COMPOSER_FEATURES.SubModelSelection]: false,
+        [COMPOSER_FEATURES.SceneHierarchySearch]: true, // Entity Search
+        [COMPOSER_FEATURES.SceneHierarchyMultiSelect]: true, // MultiSelect disabled, not sure if we will support this.
```

Comment:
We should not be turning on multi-select.

Reply:
Should I just revert the one feature or leave the file alone entirely?

Reply:
It is off now.